kubetest2/internal/deployers/eksapi/templates/unmanaged-nodegroup.yaml.template

```diff
@@ -24,7 +24,7 @@ Parameters:
   NodeDiskSize:
     Type: Number
     Description: Node disk size in gigabytes.
-    Default: 20
+    Default: 100
```

Comment:
is this to fix the issue in https://github.com/aws/aws-k8s-tester/actions/runs/10188407331/job/28184511933?pr=454#step:3:12945?

Reply:
No, it fix the issue when the cluster/nodes created by kubetest2 don't have enough storage to pull the image to the node.